crates/compiler/gen_dev/src/generic64/mod.rs

```diff
@@ -1191,14 +1198,22 @@ impl<
     fn build_eq(&mut self, dst: &Symbol, src1: &Symbol, src2: &Symbol, arg_layout: &InLayout<'a>) {
         match *arg_layout {
             single_register_int_builtins!() | Layout::BOOL => {
+                let width = match *arg_layout {
```

Comment:
I assume we will want to make this a standalone function because it will be used all over the place.

Reply:
yeah, we'll see